bin/gather_logs.sh

```diff
@@ -48,11 +44,24 @@ sort_by_last() {
 	-o 'UserKnownHostsFile /dev/null')
     logs=(/var/log /etc)
     logs+=(/var/chef/cache /var/cache/chef /opt/dell/crowbar_framework/db)
-    crowbarctl node list -U machine-install -P $CROWBAR_PASS --no-verify-ssl
 
-    for to_get in proposals roles; do
-        crowbarctl $to_get proposal list crowbar -U machine-install -P $CROWBAR_PASS --no-verify-ssl
+    mkdir nodes
+    for node in $(crowbarctl node list --no-verify-ssl --no-meta --plain); do
+        crowbarctl node show $node --no-verify-ssl --json > nodes/$node.json
```

Comment:
If I understand correctly, this will be collected by the `find` on line 73 now?

Reply:
Correct. Anything that is not a file will not end up in the tarball generated by that script and is therefore useless.

(we don't really recommend using that script since we have supportconfig, so that's why noone ever noticed)